style.css

```diff
@@ -1,190 +1,215 @@
+                    .foter_block_text_2 {
+                        display: flex;
+                        align-items: center;
+                        justify-items: center;
+                        border-bottom: 1px solid rgb(145, 145, 145);
+                        width: 400px;
+                        height: 100px;
+                        color: rgb(255, 255, 255);
+                        font-family: sans-serif;
+                        font-size: 25px;
+                    }
+                    
+                    .foter_block_button {
+                        width: 400px;
+                        height: 80px;
+                        background-color: rgb(64, 113, 206);
+                        border: 0;
+                        text-align: center;
+                        color: antiquewhite;
+                        margin-top: 25px;
+                        margin-bottom: 25px;
+                        font-size: 30px;
+                        font-family: inherit;
+                    }
+                    
+                    .foter_block_panel {
+                        display: flex;
+                        align-items: center;
+                        justify-content: space-around;
+                        width: 700px;
+                        height: 100px;
+                        margin-top: 135px;
+                        margin-bottom: 35px;
+                        color: white;
+                        background-color: rgb(1, 1, 26);
+                    }
+                    
+                    .futer_button {
+                        width: 300px;
+                        height: 50px;
+                        border: 0;
+                        color: aliceblue;
+                        font-family: "Arial Narrow Bold", sans-serif;
+                        background-color: rgb(83, 83, 184);
+                        margin-right: 20px;
+                    }
+                    
+                    .my-green {
+                        color: rgb(56, 255, 132);
+                        margin-right: 24px;
+                    }
+                    
+                    .foter_block_item-text {
+                        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+                        font-style: initial;
+                    }
+                    
+                    .color_for_link {
+                        color: aliceblue;
+                        text-decoration: none;
+                    }
+                    
+                    .color-icon {
+                        color: rgb(30, 119, 202);
+                    }
+                    
+                    .foter_block_end_img {
+                        display: flex;
+                        background-image: url(./jpg/mini_2.png);
+                        width: 30px;
+                        height: 30px;
+                    }
```

Comment:
no new line in the end of file . please add it

Reply:
done